### DISCUSSION TITLE
Is it possible to make the skybox less zoomed in?

### DISCUSSION THREAD

**Discussion by vinny-888** [google/model-viewer#4246] [Q&A]
When adding a skybox the result is very zoomed in.

Is there a way to make it less zoomed in?

**Comment by vinny-888** [comment]
Here is the result

<img width="1340" alt="image" src="https://user-images.githubusercontent.com/94944578/234125230-58b69b66-7885-4a04-9840-3c1c061ed820.png">

**Reply by elalish** [reply]
You change this by increasing the [`field-of-view`](https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-fieldOfView) of the camera (you'll also need to increase the `max-field-of-view`).

**Comment by vinny-888** [comment]
Here is the skybox image
<img width="726" alt="image" src="https://user-images.githubusercontent.com/94944578/234125172-98f0015c-e80e-443e-a548-9cfcf5f55fdd.png">

**Comment by vinny-888** [comment]
Was able to solve this with 

```
    min-camera-orbit='auto auto 5%'
    max-camera-orbit='auto auto 100%'
    min-field-of-view='90deg'
    max-field-of-view='90deg'
```
